config/routes.rb

```diff
@@ -10,16 +12,20 @@
   resources :carts
   resources :administrator
   resources :about
+  resources :sales
   resources :users do
     resources :orders
   end
 
+  get 'display/:option' => "sales#display", as: "display"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.